userguide/system.rst

```diff
@@ -1273,9 +1273,11 @@ has options to configure each %brand% :ref:`Alert`.
    Alert Settings
 
 
-Alerts are grouped by |web-ui| feature or service monitor. To adjust the
-alert level or notification frequency, choose an option from the
-drop-down menus and click :guilabel:`SAVE`.
+Alerts are grouped by |web-ui| feature or service monitor. To customize how
+important to flag an alert, use the :guilabel:`Warning Level` drop-down. To
```

Comment:
```suggestion
important an alert is flagged, use the :guilabel:`Warning Level` drop-down. To
```

Reply:
Idk if you'll like the uses of a past participle, so if you have another idea that's fine with me.